lib/hammer_cli_import/base.rb

```diff
@@ -121,6 +121,9 @@ def api_init
       # Call API. Ideally accessed via +api_call+ instance method.
       # This is supposed to be the only way to access @api.
       def api_call(resource, action, params = {}, headers = {}, dbg = false)
+        if resource == :organizations and action == :create
+          params[:organization] = {:name => params[:name]}
```

Comment:
Wouldn't it be more safe to
```ruby
params[:organization] ||= {}
params[:organization][:name] =  params[:name]
```
This way it replaces all the nested params that are possibly coming with the API request